packages/loader/container-loader/src/container.ts

```diff
@@ -623,7 +617,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                     this.logger.sendErrorEvent({
```

Comment:
(nit) I'd remove this event. IMHO it does not give anything - errors should be logged by outer scenario, and error raised should have enough info to point to this code (origin of error)